mathcomp/Makefile.common

```diff
@@ -56,11 +56,10 @@ Makefile.coq: pre-makefile $(COQPROJECT) Makefile
 
 # Test suite ---------------------------------------------------------
 
-test_suite/hierarchy_test.v: build
-	mkdir -p test_suite
-	COQBIN=$(COQBIN) ocaml ../etc/utils/hierarchy.ml -verify -R . mathcomp -lib all.all > test_suite/hierarchy_test.v
+test_suite/test_hierarchy_all.v: | build
```

Comment:
@CohenCyril I meant "order-only prerequisites" in [this documentation](https://www.gnu.org/software/make/manual/html_node/Prerequisite-Types.html). If we remove this pipe symbol, repeating `make test-suite` forces us to rerun the test suite.

Reply:
After re-reading the documentation, I think this is not the right solution. Sorry for the mess.